tests/Rule/LatteTemplatesRule/PresenterWithoutModule/Fixtures/templates/Links/default.latte

```diff
@@ -85,3 +85,7 @@
 <a href="{link this#anchor}">Link to this with anchor</a>
 <a href="{link this, foo: bar, bar: baz}">Link to this with params</a>
 <a href="{link this#anchor, foo: bar, bar: baz}">Link to this with params and anchor</a>
+
+<a href="{link create persistent1: "persistent"}">With persistent annotation</a>
```

Comment:
We need also create tests where persisten parameter is part of action e.g. `actionSomething($persistent)` <- here we want to keep persistent parameter.

Reply:
Ok. Have to check how this conflict is handled by nette

Reply:
Do you think nette get persistent parameters in link as some special parameter?

Reply:
No. But i am not sure how request is provesed when method param and presistent param conflicts.